KomaMRICore/src/simulation/SimulatorCore.jl

```diff
@@ -5,6 +5,16 @@ abstract type SpinStateRepresentation{T<:Real} end #get all available types by u
 include("Bloch/BlochSimulationMethod.jl") #Defines Bloch simulation method
 include("Bloch/BlochDictSimulationMethod.jl") #Defines BlochDict simulation method
 
+"""
+Checks if the MRD type is part of SimulationOutput
+It requires the function subtypes() which is part of InteractiveUtils
+"""
+function default_return_type()
+    types = subtypes(SimulationOutput)
+    index = findfirst(i -> (string(i) == "MRD" || string(i) == "KomaMRIIO.MRD"), types)
+    return isnothing(index) ? SigArray() : types[index].instance
+end
+
```

Comment:
remove this

Reply:
Addessed in [8ff77bb](https://github.com/cncastillo/KomaMRI.jl/commit/8ff77bb58ec13a4a60ab621f337f8f8876f0f8b5)